nvim/after/plugin/lsp.lua

```diff
@@ -0,0 +1,78 @@
+local lsp = require('lsp-zero')
+
+local lspkind = require'lspkind'
+lspkind.init()
+
+lsp.preset('recommended')
+
+-- Fix Undefined global 'vim'
+lsp.configure('sumneko_lua', {
+    settings = {
+        Lua = {
+            diagnostics = {
+                globals = { 'vim' }
+            }
+        }
+    }
+})
+
+local cmp = require('cmp')
+local cmp_select = { behavior = cmp.SelectBehavior.Select }
+
+local cmp_mappings = lsp.defaults.cmp_mappings({
+    ['<C-d>'] = cmp.mapping.scroll_docs(-4),
+    ['<C-f>'] = cmp.mapping.scroll_docs(4),
+    ['<C-Space>'] = cmp.mapping.complete(),
+    ['<C-e>'] = cmp.mapping.confirm({ select = true }),
```

Comment:
`ENTER` might needs to be disabled. I find it annoying when a new line is needed, but when the LSP has some suggestions, it will autocomplete it.